src/pages/Main/CountDown.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+class CountDown extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      hours: 80,
+      minutes: 0,
+      seconds: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.myInterval = setInterval(() => {
+      if (this.state.seconds > 0) {
+        this.setState({ seconds: this.state.seconds - 1 }, this.updateStorage);
+      } else {
+        if (this.state.minutes > 0) {
+          this.setState(
+            { minutes: this.state.minutes - 1 },
+            this.updateStorage
+          );
+          this.setState({ seconds: 59 }, this.updateStorage);
+        }
+      }
+      if (this.state.seconds === 0) {
+        if (this.state.minutes === 0) {
+          if (this.state.hours === 0) {
+            clearInterval(this.myInterval);
+          } else {
+            this.setState({ hours: this.state.hours - 1 }, this.updateStorage);
+            this.setState({ minutes: 59 }, this.updateStorage);
+            this.setState({ seconds: 59 }, this.updateStorage);
+          }
+        }
+      }
```

Comment:
```suggestion
      if (!seconds && !minutes) {
        if (hours) clearInterval(this.myInterval);
        else this.setState({ hours: hours - 1, minutes: 59, seconds: 59 }, this.updateStorage);
      }
```
- 불필요한 if 문의 중첩이 있는 듯 하여 위와 같이 정리해볼 수 있겠습니다
- setState도 객체 형태로 업데이트 할수 있는 만큼 한번에 묶어서 업데이트 해주시면 됩니다
- 구조분해 할당 습관적으로 잘 해주세요!